components/list/list-item-placement-marker.js

```diff
@@ -0,0 +1,29 @@
+import { html, LitElement } from 'lit-element/lit-element.js';
+
+class ListItemPlacementMarker extends LitElement {
+	static get properties() {
+		return {
+			color: { type: String },
+			displayed: {type: Boolean}
+		};
+	}
+
+	constructor() {
+		super();
+		this.color = "blue";
+		this.strokeWidth = 4;
+		this.radius = this.strokeWidth * 2;
+		this.height = this.radius * 2 + this.strokeWidth * 2;
+	}
+
+	render() {
+	  return this.displayed ? html`
+		<svg height="${this.height}"  width="100%">
+		  	<circle cx="${this.strokeWidth + this.radius}" cy="${this.strokeWidth + this.radius}" r="${this.radius}" stroke="${this.color}" stroke-width="${this.strokeWidth}" fill="white" />
```

Comment:
In general, we will want as much of these style-specific things to be in `static get styles()` as possible. They're easier to view and manipulate this way. See [Maya's prototype for an example](https://github.com/BrightspaceHypermediaComponents/activities/blob/majones/PROTOTYPE/dragList/components/d2l-activity-collection-editor/d2l-activity-collection-list-item.js#L358)

It's also possible to have expressions or calculated values in these styles: [Lit-element styles](https://lit-element.polymer-project.org/guide/styles#expressions)